src/components/screens/MainScreen.tsx

```diff
@@ -58,24 +42,22 @@ const bottomBlockWrapper = classnames(
 )
 
 export default function () {
+  const [, setLocation] = useLocation()
+
+  useEffect(() => {
+    const urlSearchParams = new URLSearchParams(window.location.search)
+
+    if (urlSearchParams.has('token'))
+      setLocation(`/token/${urlSearchParams.get('token')}`)
```

Comment:
https://github.com/BigWhaleLabs/ketl-landing/assets/49658988/98323511-49ea-4f50-b9f4-b0922cf6c251

We have a bug here because or this, let's fix it

Reply:
Fixed